cmd/rocsp-tool/main.go

```diff
@@ -53,18 +53,18 @@ type ProcessingSpeed struct {
 	// If using load-from-db, this limits how many items per second we
 	// scan from the DB. We might go slower than this depending on how fast
 	// we read rows from the DB, but we won't go faster. Defaults to 2000.
-	RowsPerSecond int
+	RowsPerSecond int `validate:"min=0"`
 	// If using load-from-db, this controls how many parallel requests to
 	// boulder-ca for OCSP signing we can make. Defaults to 100.
-	ParallelSigns int
+	ParallelSigns int `validate:"min=0"`
 	// If using load-from-db, the LIMIT on our scanning queries. We have to
 	// apply a limit because MariaDB will cut off our response at some
 	// threshold of total bytes transferred (1 GB by default). Defaults to 10000.
-	ScanBatchSize int
+	ScanBatchSize int `validate:"min=0"`
```

Comment:
These all have a "Defaults to" comment, which implies they shouldn't have `min=0`. Zero is fine because that means "default."

Reply:
I'm only validating that the user hasn't supplied a negative integer.

Reply:
oh right, I missed that this was min=**0**; I was thinking min=**1**. Looks good. 👍🏻